aggregate/housing_production/change_in_units.py

```diff
@@ -180,4 +186,4 @@ def change_in_units(geography: str, write_to_internal_review=False):
             "housing_production",
         )
 
-    return final
\ No newline at end of file
+    return final
```

Comment:
A lot of the changes to this script are stylistic (adding double quotes, regorganizing code into one line, etc) to follow bandit lint conventions

Reply:
yea hopefully Te using the same linter as us will remove this issue